Yosemite/Yosemite/Model/Storage/OrderItemRefund+ReadOnlyConvertible.swift

```diff
@@ -13,6 +13,7 @@ extension Storage.OrderItemRefund: ReadOnlyConvertible {
         name = orderItemRefund.name
         productID = orderItemRefund.productID
         variationID = orderItemRefund.variationID
+        refundedItemID = orderItemRefund.refundedItemID ?? 0
```

Comment:
Defaulting an ID to 0 in Storage could have... unintended consequences when this data is used. It could mean that the refund item gets mapped to some other order, or every order, or no orders... and become difficult to delete.

It's probably not a big problem with the way we use core data, and this being a new object, but have you considered any alternatives here?